src/qml/BookmarkProperties.qml

```diff
@@ -25,41 +24,45 @@ Popup {
         groupField.value = bookmarkGroup;
     }
 
+    function saveBookmark() {
+        bookmarkModel.updateBookmarkDetails(bookmarkProperties.bookmarkId, nameField.text, groupField.value)
+    }
+
     Page {
         width: parent.width
-        height: propertiesLayout.childrenRect.height + 68
         padding: 10
         header: ToolBar {
             id: toolBar
-            height: 48
 
             background: Rectangle {
                 color: "transparent"
+                height: 48
             }
 
-            Label {
-              anchors.centerIn: parent
-              leftPadding: 48
-              rightPadding: 48
-              width: parent.width - 20
-              text: qsTr('Bookmark Properties')
-              font: Theme.strongFont
-              color: Theme.mainColor
-              horizontalAlignment: Text.AlignHCenter
-              wrapMode: Text.WordWrap
-            }
+            RowLayout {
+                width: parent.width
+                height: 48
 
-            QfToolButton {
-                id: closeButton
-                anchors {
-                    top: parent.top
-                    right: parent.right
+                Label {
+                    Layout.leftMargin: 48
+                    Layout.fillWidth: true
+                    Layout.alignment: Qt.AlignVCenter
+                    text: qsTr('Bookmark Properties')
```

Comment:
Just a general question, do we know if we use Title Case or Sentence case for such buttons? At least we should make sure we are consistent.

Reply:
Good catch, sentence case.

Reply:
Oh sorry @suricactus I thought you were talking about buttons. For panel titles I'd say title case better.

Reply:
Ok, cancel editing needs to be fixed